docs/pages/develop/tools.mdx

```diff
@@ -49,6 +49,18 @@ To use EAS CLI, you need to install it globally on your local machine by running
 
 You can use `eas --help` in your terminal window to learn more about the available commands. For a complete reference, see [`eas-cli` npm page](https://www.npmjs.com/package/eas-cli).
 
+## Expo Doctor
+
+Expo Doctor is a command line tool used to diagnose issues in your Expo project. To use it, run the following command in your project's root directory:
+
+<Terminal cmd={['$ npx expo-doctor']} />
+
+This command performs checks and analyzes your project's codebase for common issues, dependency compatibility, configuration files, and the overall health of the project. Once the check is complete, Expo Doctor outputs the results.
```

Comment:
```suggestion
This command performs checks and analyzes your project's codebase for common issues in [app config](/workflow/configuration/) and **package.json** files, dependency compatibility, configuration files, and the overall health of the project. Once the check is complete, Expo Doctor outputs the results.
```

After re-thinking it, I think let's explain what those common issues related to. The CLI output you shared checks these two files.

Apologies for overlooking this in my previous suggestion.

Reply:
Yes, I like it better. Thank you